control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go

```diff
@@ -2,11 +2,12 @@ package oapi
 
 import (
 	"fmt"
-	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/konnectivity"
 	"net/url"
 	"path"
 	"strings"
 
+	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/konnectivity"
```

Comment:
nit: group them all and goimports

Reply:
What grouping do we want to use ? There is https://github.com/coreydaley/openshift-goimports which can update go files to use a standard grouping (and thus be used to enforce it in CI)